front/src/router/index.ts

```diff
@@ -298,34 +353,38 @@ const routes: Array<RouteConfig> = [
     component: Admin,
     redirect: ROUTES_PATH.ADMIN + ROUTES_PATH.USER,
     props: true,
-    children: [{
-      name: ROUTES_NAMES.ADMIN.USER,
-      path: ROUTES_PATH.ADMIN + ROUTES_PATH.USER,
-      component: UserManagement,
-      meta: {
-        needAuthentication: true,
-        hasBurgerMenu: true,
-      },
-    }, {
-      name: ROUTES_NAMES.ADMIN.GROUP,
-      path: ROUTES_PATH.ADMIN + ROUTES_PATH.GROUP,
-      component: GroupManagement,
-      props: true,
-      meta: {
-        needAuthentication: true,
-        hasBurgerMenu: true,
-      },
-      children: [{
-        name: ROUTES_NAMES.ADMIN.GROUP_DETAIL,
-        path: ROUTES_PATH.ADMIN + ROUTES_PATH.GROUP + '/:groupId',
+    children: [
+      {
+        name: ROUTES_NAMES.ADMIN.USER,
+        path: ROUTES_PATH.ADMIN + ROUTES_PATH.USER,
+        component: UserManagement,
+        meta: {
+          needAuthentication: true,
+          hasBurgerMenu: true,
+        },
+      }, {
+        name: ROUTES_NAMES.ADMIN.GROUP,
```

Comment:
We have the same bug for the admin routes when we add a group, could you manage this usecase too ?